src/sidebar/util/test/sentry-test.js

```diff
@@ -38,22 +36,15 @@ describe('sidebar/util/sentry', () => {
   });
 
   describe('init', () => {
-    beforeEach(() => {
-      fakeDocumentReferrer = sinon.stub(document, 'referrer');
-      fakeDocumentReferrer.get(() => 'https://example.com');
-
-      fakeDocumentCurrentScript = sinon.stub(document, 'currentScript');
-      fakeDocumentCurrentScript.get(() => ({
-        src: 'https://cdn.hypothes.is/hypothesis/1.123.0/build/scripts/sidebar.bundle.js',
-      }));
+    const originalURL = import.meta.url;
 
+    beforeEach(() => {
       // Reset rate limiting counters.
       sentry.reset();
     });
 
     afterEach(() => {
-      fakeDocumentCurrentScript.restore();
-      fakeDocumentReferrer.restore();
+      import.meta.url = originalURL;
```

Comment:
It turns out that the native `import.meta.url` property is, conveniently, writable. Hence we can modify it during tests.

Reply:
Interesting...